tests/serve/test_dynamo_serve.py

```
@@ -0,0 +1,241 @@
+        pytest.param("agg_router", marks=[pytest.mark.vllm, pytest.mark.gpu]),
+        pytest.param("disagg", marks=[pytest.mark.vllm, pytest.mark.gpu_2]),
+        pytest.param("disagg_router", marks=[pytest.mark.vllm, pytest.mark.gpu_2]),
+        pytest.param("multimodal_agg", marks=[pytest.mark.vllm, pytest.mark.gpu_2]),
+        pytest.param("sglang", marks=[pytest.mark.sglang, pytest.mark.gpu_2]),
+    ]
+)
+def deployment_graph_test(request):
+    """
+    Fixture that provides different deployment graph test configurations.
+    """
+    global deployment_graphs
```

Pylint (3.3.7)

[warning] Using global for 'deployment_graphs' but no assignment is done

(W0602)